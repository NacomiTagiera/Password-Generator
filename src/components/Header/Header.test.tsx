import { render, screen } from '@testing-library/react';

import { Header } from '.';

describe('Header', () => {
  it('renders correctly', () => {
    const { asFragment } = render(<Header>Test</Header>);

    expect(asFragment()).toMatchSnapshot();
  });

  it('displays its children', () => {
    render(<Header>Test</Header>);

    screen.getByText('Test');
  });
});
