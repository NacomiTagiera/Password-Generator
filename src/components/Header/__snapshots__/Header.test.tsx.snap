// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<DocumentFragment>
  <h1
    class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom css-fbyw1b-MuiTypography-root"
  >
    Test
  </h1>
</DocumentFragment>
`;
